src/posts/2021-05-21-google-io-learn-something-new.md

```diff
@@ -0,0 +1,82 @@
+---
+title: "Google I/O '21: leer eens iets nieuws"
+category: inspiration
+tags: googleio, technology, codelab, workshop
+comments: true
+author: michael
+min_read: 8
+date: 2021-05-21
+permalink: /2021/05/21/google-io-learn-something-new/
+---
+
+<!-- Section: Intro -->
+
+Het is de week van Google I/O, een week vol inspiratie waarin Google de laatste (technische) snufjes en ontwikkelingen met ons deelt. Dit is een aangewezen kans om **je eigen kennis te verbreden of nieuwe skills te ontwikkelen**.
+
+<!-- Section: Over Google I/O -->
+
+### Wat is Google I/O?
+
+[Google I/O](https://events.google.com/io/) is het grootste (3-daagse) developers event van Google, live vanuit [Mountain View (California)](<https://nl.wikipedia.org/wiki/Mountain_View_(Santa_Clara_County)>). Dit jaar voor het eerst volledig virtueel.
```

Comment:
Maybe an indication in the first sentence it's a annual event.

Reply:
Updated 👍🏽